sys/ztimer/auto_init.c

```diff
@@ -92,12 +92,15 @@ ztimer_clock_t *const ZTIMER_MSEC = &_ztimer_periph_timer_rtt_msec;
 static ztimer_convert_frac_t _ztimer_convert_frac_msec;
 ztimer_clock_t *const ZTIMER_MSEC = &_ztimer_convert_frac_msec.super.super;
 ztimer_clock_t *const ZTIMER_MSEC_BASE = &_ztimer_periph_timer_usec.super;
-#    if CONFIG_ZTIMER_USEC_BASE_FREQ < FREQ_1MHZ
-#      define ZTIMER_MSEC_CONVERT_LOWER         ZTIMER_USEC_CONVERT_LOWER
-#      define ZTIMER_MSEC_CONVERT_LOWER_FREQ    CONFIG_ZTIMER_USEC_BASE_FREQ
-#    else
+#    if CONFIG_ZTIMER_USEC_BASE_FREQ == FREQ_1MHZ
 #      define ZTIMER_MSEC_CONVERT_LOWER (ZTIMER_USEC)
 #      define ZTIMER_MSEC_CONVERT_LOWER_FREQ    FREQ_1MHZ
+#    elif CONFIG_ZTIMER_USEC_BASE_FREQ == 250000LU
+#      define ZTIMER_MSEC_CONVERT_LOWER (ZTIMER_USEC)
+#      define ZTIMER_MSEC_CONVERT_LOWER_FREQ    250000LU
+#    else
+#      define ZTIMER_MSEC_CONVERT_LOWER         ZTIMER_USEC_CONVERT_LOWER
+#      define ZTIMER_MSEC_CONVERT_LOWER_FREQ    CONFIG_ZTIMER_USEC_BASE_FREQ
```

Comment:
Actually this can probably all be replaced by:

```suggestion
#      define ZTIMER_MSEC_CONVERT_LOWER         ZTIMER_USEC_BASE
#      define ZTIMER_MSEC_CONVERT_LOWER_FREQ    CONFIG_ZTIMER_USEC_BASE_FREQ
```

Reply:
https://github.com/RIOT-OS/RIOT/blob/7c3e14ed1e9cde045492e2bed3b468180668d729/sys/ztimer/auto_init.c#L71 can be removed as well.

Reply:
Thanks for the suggestions